mono.targets

```diff
@@ -0,0 +1,64 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Temporary workarounds to allow building on Unix against MSBuild on top of Mono -->
+
+  <!--
+    Normal behavior is to blacklist assemblies that don't meet ResolveAssemblyReference's analysis of what is part of the target
+    framework profile. This causes us to fail on Mono as the framework ref folders don't match up with Windows. Setting the
+    following skips this behavior- should probably do the same in the normal targets (for Windows).
+  -->
+  <PropertyGroup>
+    <IgnoreDefaultInstalledAssemblyTables>true</IgnoreDefaultInstalledAssemblyTables>
+    <IgnoreDefaultInstalledAssemblySubsetTables>true</IgnoreDefaultInstalledAssemblySubsetTables>
+  </PropertyGroup>
+
+  <!--
+    Skipping this temporarily to focus on the failures in building the product assemblies. The error we get is as follows:
+
+     RunTestsForProject:
+      mono corerun.exe xunit.console.netcore.exe System.Xml.XPath.XmlDocument.Tests.dll  -xml testResults.xml  -notrait category=OuterLoop -notrait category=failing 
+      Cannot open assembly 'corerun.exe': No such file or directory.
```

Comment:
Yeah, just need to fix it. Wanted to filter it out temporarily so I could fix the build issues.